### DISCUSSION TITLE
What variant of  SignServer are you using? 

### DISCUSSION THREAD

**Discussion by KarolinHem** [Keyfactor/signserver-ce#32] [Polls]
For more information on the deployment options, see:

[SignServer Community](https://www.signserver.org/download/?utm_medium=website&utm_source=github&utm_campaign=community)
[SignServer Enterprise](https://www.keyfactor.com/platform/keyfactor-signserver-enterprise/?utm_medium=website&utm_source=github&utm_campaign=community)

**Comment by netmackan** [comment]
Other, as obviously I use most of them :) maybe multiple options also should be supported in the poll...
